packages/shared.test/src/stubs/objectTypeV2.ts

```diff
@@ -609,30 +609,35 @@ export const BGaoNflPlayerObjectType: ObjectTypeV2 = {
             dataType: {
               type: "string",
             },
+            rid: "rid.a.b",
```

Comment:
Nit: RIDs start with `ri.`, not `rid.`, and then have four components, so `ri.a.b.c.d`. Even better would be to use realistic ones like `ri.ontology.main.property.ab2f1665-7652-4932-bf60-0f36bf9eb55e`.

Reply:
changed to `ri.a.b.c.d` for now